content/sensu-go/5.14/getting-started/learn-sensu.md

```diff
@@ -360,15 +394,15 @@ sensuctl event info sensu-go-sandbox curl_timings --format json | jq .
 ...
 {{< /highlight >}}
 
-Because we configured a metric format, the Sensu agent is able to convert the Graphite-formatted metrics provided by the check command into a set of Sensu-formatted metrics (not shown in the output), which are then sent to the InfluxDB handler that reads Sensu-formatted metrics and converts them to a format InfluxDB accepts.
-Metric support isn't limited to just Graphite; the Sensu agent can extract metrics in multiple line protocol formats, including Nagios performance data.
+Because the check definition specified a metric format of `graphite_plaintext`, the Sensu agent will treat the output of the check command as Graphite-formatted metrics and translate them into a set of Sensu-formatted metrics (not shown in the output). These metrics are then sent to the InfluxDB handler, which reads Sensu-formatted metrics and converts them to a format InfluxDB accepts.
 
-**5. See the HTTP response code events for Nginx in [Grafana](http://localhost:4002/d/go01/sensu-go-sandbox).**
+_**NOTE**: Metric support isn't limited to Graphite! The Sensu agent can extract metrics in multiple line protocol formats, including Nagios performance data._
 
-Log in to Grafana as username: `admin` and password: `admin`.
-We should see a graph of live HTTP response codes for Nginx.
+**5. See the HTTP response code events for Nginx in Grafana.**
 
-Now if we turn Nginx off, we should see the impact in Grafana:
+Log in to [Grafana](http://localhost:4002/d/go01/sensu-go-sandbox) with username: `admin` and password: `admin`. You should see a graph of live HTTP response codes for Nginx.
```

Comment:
You still have to log in with the `admin` credentials, and Grafana still prompts you to change your password (you can click Skip)

Reply:
Is this the case with the `anonymous-grafana` branch for https://github.com/sensu/sandbox/pull/60 checked out? That PR needs to be merged before the grafana changes are expected to take effect on `master` branch.

Reply:
I forgot we hadn't merged that one -- I tried again with that branch checked out. With the branch checked out, Grafana won't load. I made a short gif to show what I see:
![2019-10-21_20-11-37](https://user-images.githubusercontent.com/19190208/67252060-5e140c80-f43f-11e9-82d9-a962d3eff6c7.gif)

Reply:
It actually does work, it's just listening on the wrong port (3000) and also caused the Sensu Dashboard to not work either.

The setup script actually changes the default from 3000 to 4000 to not conflict with the Sensu Go Dashboard here:

https://github.com/sensu/sandbox/blob/master/sensu-go/provision/setup.sh#L140-L141

But that is lost with the addition of the file in the PR https://github.com/sensu/sandbox/pull/60 as the recursive coping of the files in `/vagrant_files/etc/grafana/` clobbers the one created by the installation of Grafana. We should add the custom port line of `http_port = 4000` to the just mentioned PR and remove it from setup.sh. It is the only customization I see in setup.sh against the default config.

Also in setup.sh, line 140 has a reference to Uchiwa when that should be the Sensu Go Dashboard.

Reply:
And with all these changes, we should consider a mention somewhere in this document that previous users should do a `git pull` to get the latest updates, as we are making other changes that assume a user may re-run this at a later time.